tools/create-spec

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
+
+import argparse
+import contextlib
+import fileinput
+import json
+import re
+import sys
+
+
+DOWNLOAD_URL = 'https://github.com/cockpit-project/cockpit/releases/download/'
+
+
+def wip_version(args):
+    return 'g' in args.version
+
+
+def substitute(match, args, package):
+    variable = match.group(1)
+
+    if variable == 'VERSION':
+        return args.version
```

Comment:
It's fine, just in case you feel like some nitpicking: This could be a static dictionary with varname → value. Avoids the unsightly long if/elif ladder. There's even [string.Template](https://docs.python.org/3/library/string.html?highlight=template#string.Template) for that kind of thing. It even has custom delimiters, if you like `@` more than `$` :grin:

Reply:
Some cases could be a dictionary.  But consider that the BASE_VERSION is evaluated differently for each section...

Reply:
I've done a demo commit to try a dictionary-based approach.  TL;DR: I don't think I like it...

Reply:
Thanks, nevermind. Totally not important.